clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -63,9 +63,12 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
           LogEntry.newBuilder(
                   Payload.JsonPayload.of(
                       mapper.readValue(
-                          JsonFormat.printer().print(auditLogRequest),
+                          JsonFormat.printer()
+                              .preservingProtoFieldNames()
+                              .print(auditLogRequest.getPayload()),
                           new TypeReference<Map<String, ?>>() {})))
               .setLogName(getLogNameFromLogType(auditLogRequest.getType()))
+              .setLabels(auditLogRequest.getLabelsMap())
```

Comment:
Do we also need the operation and timestamp fields to make it identical to go cloud logging processor? https://github.com/abcxyz/lumberjack/blob/545bd8eeb449168cb2b863a3177ae1d1c5cdc3ca/clients/go/pkg/cloudlogging/processor.go#L131

Reply:
I create another bug to track it https://github.com/abcxyz/lumberjack/issues/341. Looks like it is not trivial, need to do some type conversion.

Reply:
Thank you both for catching the issue!

Can we add a comment here for why we need this?